documentation/src/main/asciidoc/topics/proc_configuring_server_fips.adoc

```diff
@@ -1,25 +1,34 @@
 [id='configuring-server-fips_{context}']
 = Configuring {brandname} Server on a system with FIPS 140-2 compliant cryptography
 
-FIPS (Federal Information Processing Standards) are standards and guidelines for US federal computer systems. Although FIPS are developed for use by the US federal government, many in the private sector voluntarily use these standards.
+FIPS (Federal Information Processing Standards) are standards and guidelines for US federal computer systems. 
+Although FIPS are developed for use by the US federal government, many in the private sector voluntarily use these standards.
 
-FIPS 140-2 defines security requirements for cryptographic modules. It is possible to configure {brandname} Server to use encryption ciphers which adhere to the FIPS 140-2 specification using alternative JDK security providers:
+FIPS 140-2 defines security requirements for cryptographic modules. 
+You can configure your {brandname} Server to use encryption ciphers that adhere to the FIPS 140-2 specification by using alternative JDK security providers.
+
+.Additional resources
 
 * link:{jdkroot}/security/pkcs11-reference-guide1.html[Java PKCS#11 cryptographic provider]
 * link:https://www.bouncycastle.org/fips_faq.html[The Legion of the Bouncy Castle cryptographic provider]
 
-== Configuring the PKCS#11 cryptographic provider
+== Configuring the PKCS11 cryptographic provider
+
+You can configure the PKCS11 cryptographic provider by specifying the PKCS11 keystore with the `SunPKCS11-NSS-FIPS` provider.
+
 
 .Prerequisites
 
-* A system in FIPS mode. Execute `fips-mode-setup --check` to ensure it is enabled.
-* A correctly initialized NSS database.
-* A JDK installation with the `java.security` file configured to enable the `SunPKCS11` provider pointing to the NSS database and the SSL provider initialized with the `SunPKCS11-nss-fips` argument.
-* A certificate installed in the NSS database.
+* Configure your system for FIPS mode. You can check if your system has FIPS Mode enabled by issuing the `fips-mode-setup --check` command in your {brandname command-line Interface (CLI)
+* Initialize an NSS database by issuing the with the `SunPKCS11-nss-fips` argument in your {brandname} CLI.
```

Comment:
Tough crowd :) ! 

Possible to change `using` to `by using`? 

_Using_ on its own can be ambiguous, as it could refer one of the following meanings:

- Use the `certutil` tool to initialize the system-wide NSS database.
- Initialize a system-wide NSS database that happens to use a `certutil` tool (specifically targeting one database configuration over another type)